scripts/chatbot.js

```diff
@@ -116,8 +117,7 @@ async function newChatbotMessage(chatbotResponse) {
 }
 
 function routeToContact() {
-    const contactElement = document.getElementById("socials-section")
-    contactElement.scrollIntoView( { behavior: "smooth", block: "start", inline: "center" } )
+    window.location.href = "contact.html"
```

Comment:
Why did you change the original smooth scrolling implementation cause I believe it was better as it allowed users to quickly navigate to the specific part of the contact page?

Reply:
That implementation scrolled downwards to the footer for contacting me. This new implementation is meant to route to the contact page instead